src/main/scala/fs2/StreamDerived.scala

```diff
@@ -57,6 +57,9 @@ private[fs2] trait StreamDerived { self: fs2.Stream.type =>
   def eval_[F[_],A](fa: F[A]): Stream[F,Nothing] =
     flatMap(eval(fa)) { _ => empty }
 
+  def infinite[F[_],W](ws: W*): Stream[F, W] =
```

Comment:
Pls call this `constant` to be consistent with existing signature in old code:

```Scala
  /**
   * The infinite `Process`, always emits `a`.
   * If for performance reasons it is good to emit `a` in chunks,
   * specify size of chunk by `chunkSize` parameter
   */
  def constant[A](a: A, chunkSize: Int = 1): Process0[A]
```

Also, just have it take a single `W`. If you want to repeat a sequence over and over, just do `emits(ws).repeat`.